extensions/amp-script/0.1/amp-script.js

```diff
@@ -25,18 +25,19 @@ import {
 import {Layout, isLayoutSizeDefined} from '../../../src/layout';
 import {Services} from '../../../src/services';
 import {UserActivationTracker} from './user-activation-tracker';
-import {calculateExtensionScriptUrl} from '../../../src/service/extension-location';
 import {cancellation} from '../../../src/error';
 import {closestAncestorElementBySelector} from '../../../src/dom';
 import {dev, user, userAssert} from '../../../src/log';
 import {dict} from '../../../src/utils/object';
 import {getElementServiceForDoc} from '../../../src/element-service';
-import {getMode} from '../../../src/mode';
 import {rewriteAttributeValue} from '../../../src/url-rewrite';
 import {startsWith} from '../../../src/string';
 import {tryParseJson} from '../../../src/json';
 import {utf8Encode} from '../../../src/utils/bytes';
 
+// TODO(friedj): before merging we need to be able to dynamically decide between .js and .max.js
```

Comment:
Would it work if this was a function?

```js
export function ampScriptWorkerJs() {
  return getMode().minified ? /** min */ : /** max */;
}
```

Reply:
I like the idea, but I don't think we could do that without it also causing both versions to be inlined

Reply:
I think Closure compiler will transform `getMode().minified` into a constant and tree shake the other branch.

Reply:
I bear good tidings! 
I tested it out by running `gulp dist` and inspecting the generated file -- it works :).